documentation/source/user_section/courses.rst

```diff
@@ -7,7 +7,8 @@ We organize a **free** in-person SCT course each year after the ISMRM conference
 
 The past courses handouts are listed below:
 
--  `SCT course (v4.2.1), London, 2020-01-21`_ \| `Video recording`_
+-  `SCT course (v5.4.0), Boston, 2021-11-16`_ \| `Video recording <https://www.youtube.com/watch?v=Pcg2ngc9hj8&list=PLJ5-Fnq9XpaXmDmtwDPycLyZoitv8PsTi&index=2>`__
+-  `SCT course (v4.2.1), London, 2020-01-21`_ \| `Video recording <https://www.youtube.com/watch?v=whbtjYNtHko>`__
```

Comment:
Note that previously, we only had a single link titled `Video recording`. If I were to add a second, then there would be a name conflict. So, instead, I changed both `Video Recording` links to be [anonymous links](https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use) instead.

See also: https://github.com/sphinx-doc/sphinx/issues/3921#issuecomment-315581557